### PR TITLE
test: Add storaged to debian-unstable image

### PR DESCRIPTION
**Pull request by mvollmer**
It's in experimental now.

### CODE REVIEW

**Comment by mvollmer** [comment]
bot: Image refresh for debian-unstable

**Comment by cockpituous** [comment]
Image creation for debian-unstable in process on host37-rack09.
Log: http://fedorapeople.org/groups/cockpit/logs/refresh-debian-unstable-1ec57c59fe560b62ecf0570602f67b5fadf26484/

**Comment by cockpituous** [comment]
Image creation failed.

**Comment by mvollmer** [comment]
bot: Image refresh for debian-unstable

**Comment by cockpituous** [comment]
Image creation for debian-unstable in process on host32-rack06.
Log: http://fedorapeople.org/groups/cockpit/logs/refresh-debian-unstable-994a7731e9a68aa5195a21d5d2c0c57c0f6d61f5/

**Comment by cockpituous** [comment]
Image creation failed.

**Comment by mvollmer** [comment]
bot: Image refresh for debian-unstable

**Comment by cockpituous** [comment]
Image creation for debian-unstable in process on host37-rack09.
Log: http://fedorapeople.org/groups/cockpit/logs/refresh-debian-unstable-d0cc73066a49c0407d7198749295483d9dd30d50/

**Comment by cockpituous** [comment]
Image creation failed.

**Comment by mvollmer** [comment]
bot: Image refresh for debian-unstable

**Comment by cockpituous** [comment]
Image creation for debian-unstable in process on host37-rack09.
Log: http://fedorapeople.org/groups/cockpit/logs/refresh-debian-unstable-c531f0565fe70cb3463d7a82509c3d25eb2e5641/

**Comment by cockpituous** [comment]
Image creation done: https://github.com/cockpituous/cockpit/commits/refresh-debian-unstable-c531f0565fe70cb3463d7a82509c3d25eb2e5641

File: test/images/scripts/debian-unstable.setup [hunk 1]
```diff
@@ -76,6 +78,10 @@ curl \
 useradd -m -U -c Administrator -G sudo -s /bin/bash admin
 echo admin:foobar | chpasswd
 
+# For storaged
+echo "deb http://ftp.uk.debian.org/debian experimental main" >/etc/apt/sources.list.d/experimental.list
```
Comment (dperpeet) [inline test/images/scripts/debian-unstable.setup]: https://wiki.debian.org/DebianExperimental#To_install_an_experimental_package recommends `deb http://httpredir.debian.org/debian experimental main` and let that pick a mirror
This change won't need an image respin.

Comment (mvollmer) [inline test/images/scripts/debian-unstable.setup]: I have just copied the url from `debian-unstable.bootstrap`, and that one is a copy of what was in the virt-builder template, I think.

We can certainly switch, in both places.

Comment (mvollmer) [inline test/images/scripts/debian-unstable.setup]: Done

**Pull request closed without merging**